src/psbt.h

```diff
@@ -578,7 +578,7 @@ bool PSBTInputSigned(const PSBTInput& input);
  * txdata should be the output of PrecomputePSBTData (which can be shared across
  * multiple SignPSBTInput calls). If it is nullptr, a dummy signature will be created.
  **/
-bool SignPSBTInput(const SigningProvider& provider, PartiallySignedTransaction& psbt, int index, const PrecomputedTransactionData* txdata, int sighash = SIGHASH_ALL, SignatureData* out_sigdata = nullptr);
+bool SignPSBTInput(const SigningProvider& provider, PartiallySignedTransaction& psbt, int index, const PrecomputedTransactionData* txdata, int sighash = SIGHASH_ALL, bool finalize = true, SignatureData* out_sigdata = nullptr);
```

Comment:
Please take more care when changing function signatures. This could silently cast a `SignatureData*` to a `bool` when merging another PR created prior to the API change.

Reply:
Changed to the last parameter